wiki/Developer-Documentation.md

```diff
@@ -125,6 +125,33 @@ and performs various checks, it may take a couple of minutes to complete. If you
 are running this from a partition that is nearly full, the test may fail. Please
 refer to `maxSpaceUtilizationThreshold` in [[Configuration Reference]].
 
+#### Writing Tests
```

Comment:
Should we state that one needs to run "make gtest" to test his/her test units? I'm assuming you omit to put this information here because ultimately we want every test to be done under google test and a simple "make test" should do it. That correct?